src/main/resources/application.properties

```diff
@@ -41,19 +39,21 @@ spring.mvc.async.request-timeout=${SYSTEM_CONNECTIONTIMEOUTMILLISECONDS:1200000}
 #spring.thymeleaf.prefix=file:/customFiles/templates/,classpath:/templates/
 #spring.thymeleaf.cache=false
 
-spring.datasource.url=jdbc:h2:file:./configs/stirling-pdf-DB-2.3.232;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
```

Comment:
We should support and still default to using a DB file for ease of adoption and use of users,
We should still allow H2 file connection and ensure this is the default type 
But still offer it to users for postgres etc